src/server/scripts/Spells/spell_paladin.cpp

```diff
@@ -463,40 +464,14 @@ class spell_pal_divine_storm_dummy : public SpellScriptLoader
         {
             PrepareSpellScript(spell_pal_divine_storm_dummy_SpellScript);
 
-        public:
-            spell_pal_divine_storm_dummy_SpellScript()
-            {
-                _targetCount = 0;
-            }
-
-        private:
-            bool Validate(SpellInfo const* /*spellInfo*/) override
-            {
-                if (!sSpellMgr->GetSpellInfo(SPELL_PALADIN_DIVINE_STORM_HEAL))
-                    return false;
-                return true;
-            }
-
-            void CountTargets(std::list<WorldObject*>& targetList)
-            {
-                _targetCount = targetList.size();
-            }
-
-            void HandleDummy(SpellEffIndex /*effIndex*/)
+            void FilterTargets(std::list<WorldObject*>& targets)
             {
-                if (!_targetCount || ! GetHitUnit())
-                    return;
-
-                int32 heal = GetEffectValue() / _targetCount;
-                GetCaster()->CastCustomSpell(GetHitUnit(), SPELL_PALADIN_DIVINE_STORM_HEAL, &heal, NULL, NULL, true);
+                Trinity::Containers::RandomResizeList(targets, 0);
```

Comment:
Yes because this spell is used only for visual but with target > 0 you have a second visual on target hit but SPELL_PALADIN_DIVINE_STORM_DAMAGE give this visual and the radius it's 16 yards